build

```diff
@@ -128,6 +131,11 @@ def compile_docker(config, target, src_dir, tgt_dir, debug=False):
     dshell('/tgt/app', '/tgt/qt/bin/qmake /src/wkhtmltopdf.pro CONFIG+=silent')
     dshell('/tgt/app', 'make install INSTALL_ROOT=/tgt/wkhtmltox')
 
+    postcompile = config['docker-targets'][target].get('postcompile', '')
+    script = os.path.abspath('scripts/%s.postcompile.sh' % postcompile)
```

Comment:
Can we put everything in an `after_compile` folder? So the key is named `after_compile` and you use `after_compile/%s.sh` over here.